test/component/QueueSpec.scala

```diff
@@ -55,11 +52,8 @@ class QueueSpec extends AnyFeatureSpec
 
   implicit val ec: ExecutionContext = Helpers.stubControllerComponents().executionContext
   override implicit lazy val app: Application = new GuiceApplicationBuilder().configure(componentTestConfigs).build()
-  private val repo = new ReactiveRepository[ClientNotification, BSONObjectID](
-    collectionName = "notifications",
-    mongo = app.injector.instanceOf[MongoDbProvider].mongo,
-    domainFormat = ClientNotification.ClientNotificationJF, ReactiveMongoFormats.objectIdFormats) {
-  }
+
+  val repo: NotificationMongoRepository = app.injector.instanceOf[NotificationMongoRepository]
```

Comment:
The best way to test the repos is to extend DefaultPlayMongoRepositorySupport[<Model>]
then override the method
override protected def repository: PlayMongoRepository[NotificationMongoRepository]  = new NotificationMongoRepository(mongoComponent)

Reply:
Please do that for all repos

Reply:
What is the benefit of using the helper trait? Is there an advantage to instantiating the repo instead of injecting it? That trait is a wrapper for the same methods I have used as far as I can see

Reply:
Does injecting like this create a db like `Test-QueueSpec` in MOngo?

Reply:
Yes. It creates a db to the exact specification set out in NotificationMongoRepository